src/components/Tabber/Tabs/Tabs.tsx

```diff
@@ -1,30 +1,33 @@
-﻿import classNames from 'classnames';
+﻿import { ReactNode } from 'react';
+
+import classNames from 'classnames';
 
 import scss from '~components/Tabber/Tabs/Tabs.module.scss';
 
 import { TSocialNetworks } from '../stores/useSocialNetworkStore.types';
 import { ITabsProps } from './Tabs.types';
 
-function Tabs(props: ITabsProps) {
-  const tabClasses = (id: string) =>
+function Tabs(props: ITabsProps): ReactNode {
+  const tabClasses = (id: string): string =>
     classNames(scss.tab, id === props.currentTab.id && scss.active);
 
-  const renderTabs = (socialNetwork: TSocialNetworks) => (
-    <div
+  const renderTabs = (socialNetwork: TSocialNetworks): ReactNode => (
+    <button
       className={tabClasses(socialNetwork.id)}
       key={`${socialNetwork.id}-${socialNetwork.name}`}
       onClick={() => props.onChangeTab(socialNetwork)}
+      type="button"
```

Comment:
```suggestion
    <button
      className={tabClasses(socialNetwork.id)}
      key={`${socialNetwork.id}-${socialNetwork.name}`}
      onClick={() => props.onChangeTab(socialNetwork)}"
```
esse type="button" não é necessário se você usa o html button ele só serve pra fazer atrocidades tipo usar div como button

Reply:
Certo, entendido.